index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head id="header">
+ 
+</head>
+<body>
+  <header>
+      <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png">
+        <nav id="nav-bar">
+            <ul>
+                <li> Features</li>
+                <li> How It works</li>
+                <li> Pricing</li>
+            </ul>
+        </nav>
+  </header>
+  <section id="new-search-bar">
+    <h2>Handcrafted, home-made masterpieces</h2>
+    <form id="form">
+        <input type="email" id="email" name="email" placeholder="Enter your email address" required>
+        <input id="submit" type="submit" value="submit">
+    </form>
+  </section>
+  <section id="premium">
+    <div>
+        <h2>Premium Materials</h2>
+        <p>Our trombones use the shiniest brass which is sourced locally.This will increase 
+            the longevity of your purchase.</p>
+    </div>
+  </section>
+  <section id="shipping">
+    <div>
+        <h2>Fast Shipping</h2>
+        <p>We make sure you recieve your trombone as soon as we have finished making it.
+             We also provide free returns if you are not satisfied.</p>
+    </div>
+  </section>
+  <section id="quality">
+    <div>
+        <h2>Quality Assurance</h2>
+        <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+    </div>
+  </section>
+
+  <section id="youtube-video">
+    <iframe id="video" height="400" allowfullscreen src="https://www.youtube-nocookie.com/embed/y8Yv4pnO7qc?rel=0&controls=0&showinfo=0"></iframe>
+  </section>
+
+  <section>
+    <div id="tonor">
+        <div>TENOR TROMBONE</div>
+        <h2>$600</h2>
+        <ol>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum</li>
+            <li>Lorem ipsum dolor</li>
+            <li>Lorem ipsum</li>
+        </ol>
+        <button>Select</button>
+    </div>
+    <div id="bass">
+        <div>BASS TROMBONE</div>
+        <h2>$900</h2>
+        <ol>
+            //<li>Lorem ipsum</li>
+            //<li>Lorem ipsum</li>
+            //<li>Lorem ipsum dolor</li>
+            //<li>Lorem ipsum</li>
```

Comment:
- [ ] Please edit your comment as suggested by the previous reviewers. To edit your comment please click on the three dots `...` at the top corner of the comment container. See image below for a guide 👇🏾 

![image](https://github.com/the-faizmohammad/project-review/assets/84156714/53415e61-4261-45c9-9951-5e727cb43df9)

Then make your comment more expressive as suggested.